src/qmapshack/realtime/ais/CRtAisInfo.cpp

```diff
@@ -0,0 +1,591 @@
+/**********************************************************************************************
+    Copyright (C) 2018 Oliver Eichler <oliver.eichler@gmx.de>
```

Comment:
Don't be shy. Don't hide yourself. You had the work, you deserve the copyright.

Applies to all the other files, too.

Reply:
Came with the copy/paste from opensky and gpstether files, didn't really pay much attention to it :)